src/alpaka/Makefile

```diff
@@ -0,0 +1,286 @@
+TARGETS = alpaka 
```

Comment:
This file looks mostly a copy from the pixel-standalone. Do I guess correctly that this is a work in progress and will be updated to be part of the build system of this repo at some point soon?

Reply:
yes, precisely